core/storage/exploration/gae_models.py

```diff
@@ -239,6 +239,61 @@ def has_reference_to_user_id(cls, unused_user_id):
         return False
 
 
+class ExplorationMathRichTextInfoModel(base_models.BaseModel):
+    """Temporary Storage model for storing information useful while generating
+    images for math rich-text components in explorations.
+
+    The id of each instance is the id of the corresponding exploration.
+    """
+
+    # A boolean which indicates whether the exploration requires images to be
+    # generated and saved for the math rich-text components. If this field is
+    # False, we will need to generate math rich-text component images for the
+    # exploration. The field will be true only if for each math rich-text
+    # components there is a valid image stored in the datastore.
+    math_images_generation_required = ndb.BooleanProperty(
+        indexed=True, default=True)
+    # Approximate maximum size of Math rich-text components SVG images that
+    # would be generated for the exploration according to the length of
+    # raw_latex value.
+    estimated_max_size_of_images_in_bytes = ndb.IntegerProperty(
+        indexed=True, default=0)
+    # Set of latex values from all the math-rich text components of the
```

Comment:
Be clearer about the internal structure of this property.

Is it a list of strings? If so then why not ndb.StringProperty(repeated=True)?

Reply:
done, changed to StringProperty